README.md

```diff
@@ -389,4 +395,13 @@ Disconnect from the ledger and stop listening for events.
 ### "incoming:ssp:<token>"
 [IncomingTransfer](https://github.com/interledger/rfcs/blob/master/0004-ledger-plugin-interface/0004-ledger-plugin-interface.md#incomingtransfer) from the ledger plugin and the fulfillment string for a specific token
 
-**Kind**: event emitted by <code>[Receiver](#module_Receiver)</code>
+**Kind**: event emitted by <code>[Receiver](#module_Receiver)</code>  
+<a name="module_Receiver..reviewPaymentCallback"></a>
+
+### Receiver~reviewPaymentCallback ⇒ <code>Promise.&lt;null&gt;</code>
+**Kind**: inner typedef of <code>[Receiver](#module_Receiver)</code>  
+**Returns**: <code>Promise.&lt;null&gt;</code> - cancels the payment if it rejects.  
+
+| Param | Type | Description |
+| --- | --- | --- |
+| request | <code>PaymentRequest</code> | payment request object for the payment being reviewed |
```

Comment:
I'm confused - this says that the `reviewPayment` callback gets the payment request, but in the code it seems to get the transfer.

I'd probably pass `(request, transfer)` with the rationale that 90% of the time you don't want to parse the transfer yourself, but sometimes you may want to dig deeper.

We should think about a different name for the payment request. That name is great for IPR, but since it's the exact same string/object between KEP and IPR, it should probably have a name that works for both?